data/notices.json

```diff
@@ -283,6 +283,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "ecs breaking change renaming prop in ContainerImage.fromEcrRepository",
+      "issueNumber": 27240,
+      "overview": "Renaming a prop in ContainerImage.fromEcrRepository function introduces a breaking change, not for typescript users, but for other languages that require specifying the property name when calling the function.",
```

Comment:
Please be specific about which environment.

My guess is: only Python when using keyword parameters.